### PR TITLE
Add a new eksportisto deployment.

### PR DESCRIPTION
**Pull request by pedro-clabs**
### Description

Re-deploy eksportisto 14 with fixes simply as eksportisto (suffixes are being dropped).

### Tested

Succesfully ran an eksportisto deployment with --helmdryrun.

### Related issues

- https://github.com/celo-org/eksportisto/issues/81

### Backwards compatibility

Backwards compatible -- we're adding a new deployment, and not removing the existing 11 and 14 ones just yet.

### CODE REVIEW

File: .env.rc1 [hunk 1]
```diff
@@ -250,10 +250,15 @@ TRANSACTION_METRICS_EXPORTER_DOCKER_IMAGE_TAG="transaction-metrics-exporter-f4a5
...
+# EKSPORTISTO_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-testnet/eksportisto"
+# EKSPORTISTO_DOCKER_IMAGE_TAG="6773c9f93d715fcd0d8437dc9b5d1664f443be5a"
+# EKSPORTISTO_SUFFIX='14'
+# EKSPORTISTO_CHART_VERSION=2
+
 EKSPORTISTO_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-testnet/eksportisto"
-EKSPORTISTO_DOCKER_IMAGE_TAG="6773c9f93d715fcd0d8437dc9b5d1664f443be5a"
-EKSPORTISTO_SUFFIX='14'
-EKSPORTISTO_CHART_VERSION=2
+EKSPORTISTO_DOCKER_IMAGE_TAG="bb7e71f18c014eb2e6fbd9bf3d225839bc8e76f0"
```
Comment (ghost) [inline .env.rc1:259]: where is the image tag from?

File: packages/celotool/src/lib/eksportisto.ts [hunk 2]
```diff
@@ -19,10 +19,19 @@ const yaml = require('js-yaml')
 const chartDirForVersion: Record<number, string> = {
   1: '../helm-charts/eksportisto_11',
   2: '../helm-charts/eksportisto-2.0',
+  3: '../helm-charts/eksportisto',
+}
+
+function baseName(suffix: string) {
+  if (suffix.length > 0) {
```
Comment (ghost) [inline packages/celotool/src/lib/eksportisto.ts:26]: nice - extra nice in the separate function

File: .env.rc1 (hunk 1 above)
Comment (pedro-clabs) [reply .env.rc1:259]: From the container repository.

File: packages/helm-charts/eksportisto/templates/configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        enabled: true
+        startBlock: 0
+        batchSize: 40000
+        tipBuffer: 3
+        sleepIntervalMilliseconds: 2000
+      chainTip:
+        enabled: true
+
+    indexer:
+      concurrency: 10
```
Comment (ghost) [inline packages/helm-charts/eksportisto/templates/configmap.yaml:29]: Do we want to add `mode: both` here for completeness? Maybe cleaner than defaulting to it: https://github.com/celo-org/eksportisto/blob/master/config.yaml.example#L25

Comment (pedro-clabs) [reply packages/helm-charts/eksportisto/templates/configmap.yaml:29]: Good point. Done!

**Review by ghost — APPROVED** [review]

**Pull request merged**